locales/en.json

```diff
@@ -1650,6 +1652,11 @@
         "title": "Minting NFT Collection",
         "submit": "Minting NFT Collection..."
       },
+      "drop": {
+        "confirm": "Teleport Required Amount",
+        "action": "Claim free",
+        "item": "NFT"
```

Comment:
already exist

Reply:
wdym

Reply:
value `NFT` already exist in translation file no need to duplicate another key value

Reply:
fixed pls review